File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] word = {"aya", "ye", "woo", "ma"};
        for (String b : babbling) {
            for (String w : word) {
                if (!b.contains(w + w)) {
                    b = b.replace(w, " ");
                }
            }
            if (b.strip().length() == 0) {
                answer += 1;
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    answer = 0
    for b in babbling:
        for w in ["aya", "ye", "woo", "ma"]:
            if w * 2 not in b:
                b = b.replace(w, " ")
        if len(b.strip()) == 0:
            answer += 1
    return answer